src/sql/magic_cmd.py

```diff
@@ -65,8 +66,95 @@ def execute(self, line="", cell="", local_ns=None):
+            )
+            parser.add_argument(
+                "-g",
+                "--greater",
+                type=str,
+                help="Greater than a certain number.",
+                required=False,
+            )
+            parser.add_argument(
+                "-goe",
+                "--greater-or-equal",
+                type=str,
+                help="Greater or equal than a certain number.",
+                required=False,
+            )
+            parser.add_argument(
+                "-l",
+                "--less-than",
+                type=str,
+                help="Less than a certain number.",
+                required=False,
+            )
+            parser.add_argument(
+                "-loe",
+                "--less-than-or-equal",
+                type=str,
+                help="Less than or equal to a certain number.",
+                required=False,
+            )
+            parser.add_argument(
+                "-nn",
+                "--no-nulls",
+                help="Returns rows in specified column that are not null.",
+                action="store_true",
+            )
+
+            args = parser.parse_args(others)
+            if args.greater and args.greater_or_equal:
+                return ValueError(
+                    "You cannot use both greater and greater "
+                    "than or equal to arguments at the same time."
+                )
+            elif args.less_than and args.less_than_or_equal:
+                return ValueError(
+                    "You cannot use both less and less than "
+                    "or equal to arguments at the same time."
+                )
+
+            query = construct_string_query(args)
+            conn = sql.connection.Connection.current.session
+            res = conn.execute(query).fetchall()
+            if args.no_nulls and len(res) > 0:
+                raise ValueError(
+                    "Specified column {} has null values present.".format(args.column)
+                )
+            return res
         else:
             raise UsageError(
                 f"%sqlcmd has no command: {cmd_name!r}. "
                 "Valid commands are: 'tables', 'columns'"
             )
+
+
+def construct_string_query(args):
+    base_query = select("*").from_(args.table)
+    print(base_query)
+    if args.greater:
+        where = condition(args.column + ">" + args.greater)
+        base_query = base_query.where(where)
+    if args.greater_or_equal:
+        where = condition(args.column + ">=" + args.greater_or_equal)
+        base_query = base_query.where(where)
+    if args.less_than:
+        where = condition(args.column + "<" + args.less_than)
+        base_query = base_query.where(where)
+    if args.less_than_or_equal:
+        where = condition(args.column + "<=" + args.less_than_or_equal)
+        base_query = base_query.where(where)
+    base_query = base_query.sql()
```

Comment:
Probably can move those line together
```
    base_query = base_query.sql()
    base_query = base_query.where(where)

```

Reply:
We also need to test the compatibility of `base_query` on different databases, let's move this test case into integration test